zed_components/src/zed_camera/src/zed_camera_component.cpp

```diff
@@ -803,6 +803,7 @@ void ZedCamera::getGeneralParams()
       " * Camera framerate: ");
   }
   getParam("general.gpu_id", mGpuId, mGpuId, " * GPU ID: ");
+  getParam("general.grab_async", mAsyncImageRetrieval, mAsyncImageRetrieval, " * Asynchronous grab: ");
```

Comment:
Can you call the YAML parameter with the same name as the ZED SDK parameter?
It will be easier for costumers that work with both the SDK and the wrapper to associate them

Reply:
Updated in [29aa15f](https://github.com/stereolabs/zed-ros2-wrapper/pull/299/commits/29aa15f5b26dc62d835081e1cfc14409ee8fdd35)